### DISCUSSION TITLE
Videos on how to image bees in 2D & 3D

### DISCUSSION THREAD

**Discussion by seltmann** [Big-Bee-Network/Big-Bee-community-discussion#7] [General]
A big part of the Big-Bee project is taking hundreds of thousands of images of bees from around the world. We are doing this with the help of [Macroscopic Solutions](https://macroscopicsolutions.com/?gclid=Cj0KCQjwma6TBhDIARIsAOKuANymFGESwUKuhh8uCE11-7O2mgBeR1F5LfQi07G-SQSlxf3lxEMYfvgaAibsEALw_wcB) focus stacking systems.
![Companyweblogosmall-blue](https://user-images.githubusercontent.com/1044474/165992686-2ce7d499-2142-49d1-9046-0c8d66939adf.png)
Macroscopic Solutions has pushed the hardware and software for creating 3D photogrammetric images of bees for this project and Big-Bee imagers are testing the methods and system daily. 

If you are interested in online training for imaging bees using the Macropod 3D, you can find many videos online at [https://macroscopicsolutions.com/video-tutorial-big-bee-tcn/](https://macroscopicsolutions.com/video-tutorial-big-bee-tcn/) and the [Macroscopic Solutions YouTube](https://www.youtube.com/channel/UCXhbJ-JirA3-J7FihjXtUyg) channel

**Comment by MacroscopicSolutions** [comment]
Thank you Katja, any outreach is extremely helpful. Really grateful to you! Have the updates solved your computer problem? 

Mark R Smith
iPhone Mail
Macroscopic Solutions, LLC
***@***.***
(410)870-5566
